tracecat/actions/integrations/iam/okta.py

```diff
@@ -28,6 +28,36 @@
 """
 
 
+@registry.register(
+    default_title="Find Okta User",
+    description="Find an Okta user by login username or email",
+    display_group="Okta",
+    namespace="integrations.okta",
+    secrets=[okta_secret],
+)
+async def find_okta_user(
+    username_or_email: Annotated[
+        str,
+        Field(..., description="Login username or e-mail to find"),
+    ],
+) -> list:
+    api_token = os.getenv("OKTA_API_TOKEN")
+    base_url = os.getenv("OKTA_BASE_URL")
+    headers = {
+        "Authorization": f"SSWS {api_token}",
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+    }
+
+    async with httpx.AsyncClient() as client:
```

Comment:
Could we extract this into a function outside the UDF that returns the preconfigured client? I see you have a similar setup between the other okta integrations too.

There are some examples of this under `integrations/enrichment`